README.md

```diff
@@ -821,6 +821,7 @@ Clients for commercial social platforms that had their API access cut off in a w
 - [Emulsion](https://github.com/lainsce/emulsion) - Color palette manager `#vala` `#gtk4` `#libadwaita`.
 - [Eyedropper](https://apps.gnome.org/Eyedropper) - Color picker and formatter `#rust` `#gtk4` `#libadwaita` `#gnome`.
 - [Harvey](https://github.com/danrabbit/harvey) - Color contrast calculator `#vala` `#gtk4` `#granite`.
+- [Iconic](https://github.com/youpie/Iconic) - Custom folder icon creator `#rust` `#gtk4` `libadwaita`.
```

Comment:
```suggestion
- [Iconic](https://flathub.org/apps/nl.emphisia.icon) - Application to create folder icons and add them on top of folders in GNOME `#rust` `#gtk4` `libadwaita`.
```
I suggest using the Flathub link since it has more screenshots. I also suggest a modification of the description, feel free to change it.